app/assets/stylesheets/alchemy/icons.scss

```diff
@@ -77,7 +77,7 @@
     background-position: -256px -72px;
   }
 
-  &.delete_page {
+  &.delete_page, &.delete_node {
```

Comment:
Selector `delete_page` should be written in lowercase with hyphens<br>Selector `delete_node` should be written in lowercase with hyphens<br>Each selector in a comma sequence should be on its own single line